src/utils/country/index.ts

```diff
@@ -50,5 +64,19 @@ export const isEEACountry = (country: Country) => {
   ].includes(code);
 };
 
-export const associatedCountryFlagForCountryCode = (countryCode: CountryCode) =>
-  mapCountryFlag[countryCode];
+export const associatedCountryFlagForCountryCode = (
+  alphaCode: Alpha2CountryCode
+) => mapCountryFlag[alphaCode];
+
+export interface Country {
+  name: string;
+  code: Alpha2CountryCode;
+}
+
+export const getAllCountries = (language: languages = 'en'): Country[] =>
+  allCountryCodes.map((code: Alpha2CountryCode) => ({
+    name: localeData[language].countries[code],
+    code,
+  }));
+
+export const allCountries = getAllCountries();
```

Comment:
I can see that you want to do that for backward compatibility but maybe it's best not to have such legacies API lying around? Or maybe it's a good candidate to use the `@deprecated` directive on JSDoc (although we're not really using JSDoc 🙄)

Reply:
nice idea! I just added it